src/utils/interactive.ts

```diff
@@ -22,6 +22,13 @@ function onPointerDown(event: PointerEvent): void {
   event.preventDefault();
 }
 
+const nonBubblingWhenDisabledMouseEvents = ["mousedown", "mouseup", "click"];
```

Comment:
Do we need to worry about pointer events as well?

Reply:
Not according to the [intended spec changes](https://github.com/whatwg/html/issues/5886).